www/getting-started.rst

```diff
@@ -2,151 +2,80 @@
-of the packages such as *Python(x,y)* mentioned in :doc:`install` also
-offer an integrated scientific development environment.
+For testing the SciPy Library and Matplotlib, here's a fun Easter Egg::
 
-Neither SciPy nor NumPy provide plotting functions. There are several
-plotting packages available for Python, the most commonly used one being
-matplotlib_.
+    import scipy.misc
+    import matplotlib.pyplot as plt
 
-Learning to work with SciPy
----------------------------
+    face=scipy.misc.face()
+    plt.imshow(face)
+    plt.show()
 
-To learn more about the Python language, the `official Python tutorial`_ is
-an excellent way to become familiar with the Python syntax and objects.
 
-One way of getting a handle on the scientific computation tools in Python
-is to take a look at the following online resources:
+Start Learning
+==============
 
-- `Python Scientific Lecture Notes <http://www.scipy-lectures.org/index.html>`_
+Each package has official tutorials:
 
 - `NumPy User Guide <http://www.numpy.org/devdocs/user/>`_
 
 - `SciPy Tutorial <http://docs.scipy.org/doc/scipy/reference/tutorial/index.html>`_
-  contains examples for each submodule in the SciPy library
 
-- `Matplotlib beginner's guide <http://matplotlib.org/users/beginner.html>`__
+- `Matplotlib beginner's guide <http://matplotlib.org/users/beginner.html>`_
 
-- `Pandas tutorials <http://pandas.pydata.org/pandas-docs/stable/tutorials.html>`__
+- `Pandas tutorials <http://pandas.pydata.org/pandas-docs/stable/tutorials.html>`_
 
-- `Sympy tutorial <http://docs.sympy.org/latest/tutorial/>`__
+- `Sympy tutorial <http://docs.sympy.org/latest/tutorial/>`_
 
-In addition, a number of books have been written on numerical
-computation in Python, see for example a
-`Google search on books related to SciPy <http://www.google.com/search?q=scipy+scientific+python&tbm=bks>`__.
+Additional outside tutorials exist, such as the `Scipy Lecture Notes <http://scipy-lectures.org/index.html>`_ or `Elegant SciPy <https://github.com/elegant-scipy/notebooks>`_ .
+ 
+But the best way to learn is to start coding. 
 
+Stuck? Need Help? 
+=================
 
-An example session
-##################
+Getting errors that you can't figure out? 
 
-To give a simple example of typical interactive use, we find and plot
-the maximum of a Bessel function. If you have worked with numerical
-computation environments before, what follows looks very familiar.
+Start by looking at the error message.  Yes, error messages are often intimidating and filled with technical detail. However, they can often help pinpoint the exact location in code where things go wrong. This is often most of the battle.
 
-This assumes you have installed the SciPy stack, for example following
-the instructions in :doc:`install`.
+Unsure of how to use a particular function?  In Jupyter and the IPython shell, call up documentation with::
 
-.. sourcecode:: ipython
+    import numpy as np
+    help(np.linspace)
```

Comment:
I think `help` can be left out. The most common spelling is `np.linspace?`. And adding `np.linspace??` for viewing the source would be nice perhaps (or maybe it's just me using that all the time?).

Reply:
now have ``??`` and left out `help` on https://github.com/scipy/scipy.org/pull/312/commits/8080d86ae34516787094727b952a1f691195d233